dcp-0010/dcp-0010.mediawiki

```diff
@@ -0,0 +1,448 @@
+
+The following modified rules MUST be enforced:
+
+* The input value of the one and only input of the coinbase transaction MUST commit to the modified PoW subsidy proportion
+* The total of the output values of the coinbase transaction MUST NOT exceed the modified PoW subsidy proportion plus the total transaction fees
+* The input value of the stakebase of vote transactions MUST commit to the modified single vote subsidy proportion
+* The proportional payouts of vote transactions MUST be calculated based on the modified single vote subsidy proportion
+* The total of the output values of vote transactions MUST NOT exceed modified single vote subsidy proportion
+
+==Rationale==
+
+Proof of Work is intended to provide several key properties to Decred, some of
+which are:
+
+* Unambiguous determination of the exact contents of a block
+* Unforgeable historical timestamps
+* A source of entropy
+* Significantly increased cost to conduct a wide variety of attacks
+* Protection against data manipulation and double spends
+* Contributes to unforgeable costliness and irreversibility
+* Fair coin distribution
+
+While PoW has worked well for almost all of these properties, unfortunately
+extensive analysis has shown that the expected results of a fair coin
+distribution has not been achieved.
+
+A commonly overlooked aspect of PoW mining is that it has to behave similarly to
+gold mining in order for it to achieve the expected results of a fair coin
+distribution.  That is the reason the process is referred to as mining.
+
+More concretely, the price is expected to track the cost of production such
+that, as the profitability goes up, the incentives for competition also go up
+which leads to more participants (hash power) and thus profitability reduces
+accordingly.
+
+That expected result has not happened in Decred because there is an effective
+monopoly on the ASICs which has led to Decred significantly overpaying for what
+PoW provides.  The result of that has been a high degree of centralization of
+the coins.
+
+The most expedient approach to deal with Decred overpaying for PoW is to simply
+reduce the PoW subsidy from 60% to 10% of the block subsidy.
+
+This way, Decred retains the positive parts of PoW, like it being an entropy
+source and providing unforgeable costliness and irreversibility, without
+requiring substantial consensus work.
+
+===Alternative Approaches===
+
+Some other approaches that were considered and deemed unsuitable at the current
+time are:
+
+'''Change the PoW algorithm'''
+
+* It does not solve the underlying issue of Decred overpaying for PoW
+* It would require 6-12 months of research and debate to do properly
+* The same issue is likely to emerge with any new algorithm due to the first point
+
+'''Switch to pure PoS'''
+
+* This would require deep consensus changes and take 12+ months to execute
+* There are several attack vectors present with pure PoS that are difficult to solve
```

Comment:
* And a high likelihood of creating a myriad of new problems and unforeseen consequences that will likely require further tuning in subsequent DCPs (IMO, just commentary.)

Reply:
I originally had something very similar, as I also expect that would the case, but I decided to take it out since I felt it was more like conjecture (albeit educated) versus everything else that is grounded in known reality.

Reply:
Fair enough.  I just mean that from an engineering perspective, this being a more drastic and fundamental change has a very high chance of going sideways

Reply:
One of the immediately obvious complications is that PoW miners create block templates (and therefore determine the contents of the block, the block interval based on the difficulty, etc.).  Switching to pure PoS would require determining which PoS voter(s) create the block and how.

Reply:
Indeed.  I could create an entire list of things that would need to be solved, but I figured it was sufficiently covered by the "deep consensus changes" bullet point.